cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -45,7 +52,23 @@ def addUserInfo(self, user_info):
             },
         )
 
-        return response['ResponseMetadata']['HTTPStatusCode']
+        # Add the user to the original table
+        new_table_response = self.users.put_item(
+            Item={
+                'username': user_info['username'],
+                'register_time': str(timestamp),
+                'firstName': user_info['firstName'],
+                'lastName': user_info['lastName'],
+                'Gender': user_info['Gender'],
+                'DOB': user_info['DOB'],
+                'Grad_date': user_info['Grad_Date'],
+                'Major': user_info['Major'],
+                'Minor': user_info.get('Minor', [])
```

Comment:
We have an awesome opportunity to change our item structure here, and I suggest we take advantage of that while we can. Couple suggestions:

1. Lets make sure that we have a standard format for key casing. We have a mixture of camelCase, PascalCase, and snake_case. Any of the above is fine but we should pick a single standard.
2. Id suggest changing "major" and "minor" to their plural forms since they will be arrays. 
3. we might want to change grad_date to grad_semester, and we will want to do that conversion for whatever the frontend sends to the backend here. e.g. if the FE still sends a date, we should convert the date to a semester


Lastly, I would suggest that we use dynamodb json rather than standard json for putting our items here, because it will allow us to create strongly typed data. This will come in handy for serializing things like arrays and string sets. See https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Client.put_item for more details